test/test_replaygain.py

```diff
@@ -132,6 +141,20 @@ def test_cli_saves_album_gain_to_file(self):
 class ReplayGainGstCliTest(ReplayGainCliTestBase, unittest.TestCase):
     backend = u'gstreamer'
 
+    def setUp(self):
+        try:
+            # Check if required plugins can be loaded by instantiating a
+            # GStreamerBackend (via its .__init__).
+            config['replaygain']['targetlevel'] = 89
+            GStreamerBackend(config['replaygain'], None)
+        except FatalReplayGainError as e:
+            # Skip the test if plugins could not be loaded.
+            if str(e) == "Failed to load required GStreamer plugins":
```

Comment:
Sounds good. Yes, feel free to add a subclass—but I totally understand the instinct to use a light touch, too.